lib/browser.js

```diff
@@ -23,13 +25,30 @@ function nanoHtmlCreateElement (tag, props, children) {
     delete props.namespace
   }
 
+  // If `is` prop set then register custom element
+  if (props.is) {
+    if (typeof props.is === 'function') {
+      if (!customElements.get(props.is.tagName)) {
+        customElements.define(props.is.tagName, props.is, props.is.options)
```

Comment:
I'm not feeling great about this line - the spec really just wants you to register elements once. This would register an element on every call.

Reply:
It only registers if it isn't register yet `if (!customElements.get(props.is.tagName))`